spec/lib/salus/processor_spec.rb

```diff
@@ -7,6 +7,10 @@ def remove_key(json_string, key = 'running_time')
     json['scans'].each do |scanner, _|
       json['scans'][scanner].delete(key)
     end
+    # Avoid comparing relative and absolute file:///
+    json['config']['report_uris'].each_with_index do |endpoint, index|
```

Comment:
Is it possible that the json has no `config` key, or `json['config']` has no 'report_uris' keys?

Reply:
I can add a dig, to future proof it in case any adds some specs fixtures that lack a report endpoint.  That's unlikely as the method here is used to compare the output vs the expected output but future proofing it sounds good.

Reply:
Got it, no need to add the dig then.